mw-webapp/src/logic/wayPage/goalMetricsBlock/GoalMetricItem.tsx

```diff
@@ -44,19 +44,41 @@ export const GoalMetricItem = (props: SingleGoalMetricProps) => {
     ? `Done date ${DateUtils.getShortISODateValue(props.metric.doneDate)}`
     : "Not finished yet...";
 
+  /**
+   * Set metric not completed
+   */
+  const onOk = () => {
+    props.updateMetric({...props.metric, isDone: !props.metric.isDone, doneDate: new Date()});
```

Comment:
by name looks like this function should not toggle the "isDone" field, but set it to false. 

```suggestion
    props.updateMetric({...props.metric, isDone: false, doneDate: new Date()});
```